Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -236,6 +238,22 @@ export const projectDispatcher = () => {
     }
   );
 
+  const forceMigrate = async (projectId: string) => {
+    if (
+      await OpenConfirmModal(
+        formatMessage('Convert your project to the latest format'),
+        formatMessage(
+          'This project was created in an older version of Composer. To open this project in Composer 2.0, we must copy your project and convert it to the latest format. Your original project will not be changed.'
```

Comment:
How about 'To open this project in this version of Composer ...'

This way we don't have to think about the version number stuff in the message.